lib/application/coordinator/routes.dart

```diff
@@ -52,6 +52,33 @@ class ProgressPath extends HomePath {
   String get formattedPath => '/$name';
 }
 
+//
+// Collection Details
+//
+class CollectionDetailsPath extends AppPath {
+  CollectionDetailsPath(this.collectionId);
+  static const name = 'collection_details';
```

Comment:
What do you think about creating a routes file which holds all routes strings?

Reply:
I don't think it's useful, as they are only used here, and they are already in their own routes